ppml/base/attestation.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -x
+
+export ATTESTATION_URL=your_attestation_url
+export ATTESTATION_TYPE=your_attestation_type
+export ATTESTATION_ID=your_app_id
+export ATTESTATION_KEY=your_api_key
+export ATTESTATION_CHALLENGE=your_attestaion_challege
+export ATTESTATION_POLICYID=your_policy_id
```

Comment:
PolicyID will not be passed by hand.
It will be uploaded to runtime driver/executors as k8s secret.